static/app/views/dashboards/widgets/lineChartWidget/formatYAxisValue.tsx

```diff
@@ -0,0 +1,36 @@
+import {formatBytesBase2} from 'sentry/utils/bytes/formatBytesBase2';
+import {formatBytesBase10} from 'sentry/utils/bytes/formatBytesBase10';
+import {
+  ABYTE_UNITS,
+  DURATION_UNITS,
+  SIZE_UNITS,
+} from 'sentry/utils/discover/fieldRenderers';
+import type {RateUnit} from 'sentry/utils/discover/fields';
+import getDuration from 'sentry/utils/duration/getDuration';
+import {formatAbbreviatedNumber, formatRate} from 'sentry/utils/formatters';
+import {formatPercentage} from 'sentry/utils/number/formatPercentage';
+
+export function formatYAxisValue(value: number, type: string, unit?: string): string {
+  switch (type) {
+    case 'integer':
+      return formatAbbreviatedNumber(value);
+    case 'number':
+      return value.toLocaleString();
+    case 'percentage':
+      return formatPercentage(value, 2);
```

Comment:
I remember we had this issue come up https://github.com/getsentry/sentry/issues/57352, just want to avoid recreating the same issue

Reply:
Oh great catch, thanks!